packages/farcaster-js-neynar/CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+## Setup
+
+```sh
+# install dependencies
+yarn
+```
+
+## Running Tests
+
+Integration tests require the following environment variables be set:
+
+```
+INTEGRATION_TEST_NEYNAR_API_KEY - required for v1 & v2 testing
+INTEGRATION_TEST_NEYNAR_SIGNER_FID - required for v2
+INTEGRATION_TEST_NEYNAR_SIGNER_UUID - required for v2
+INTEGRATION_TEST_NEYNAR_SIGNER_PUBLIC_KEY: required for v2
+```
+
+Run Tests
+
+```sh
+# running from project root...
+yarn workspace @standard-crypto/farcaster-js-neynar test
+
+# ...or from from this package's directory
+yarn test
+```
+
+### Generating a Signer
+
+Generating a signer requires the following environments variable be set:
+
+```
+INTEGRATION_TEST_NEYNAR_API_KEY
+SIGNER_USER_MNEMONIC
+```
+
+Generate Signer
+
+```sh
+# running from project root...
+yarn workspace @standard-crypto/farcaster-js-neynar signer
+
+# ...or from from this package's directory
+yarn signer
+
+  NeynarAPI
+    Create/Register a signer for testing
+      ✔ can create a signer (475ms)
+      ✔ can generate a signature (78ms)
+Set env var INTEGRATION_TEST_NEYNAR_SIGNER_UUID=<signer uuid>
+Set env var INTEGRATION_TEST_NEYNAR_SIGNER_PUBLIC_KEY=<signer public key>
+Open url farcaster://signed-key-request?token=<token> on a logged in ios device to approve signer
+If using an android device, use url https://client.warpcast.com/deeplinks/signed-key-request?token=<token>
+      ✔ can register a signer (834ms)
+
+```
```

Comment:
Running a test is a weird way to prep some credentials required to run the other tests.

Do you think this would be better broken out into a `scripts/` directory or something similar? Interested to hear your thoughts, I can be convinced either way.

Happy to make this a follow-up to this PR as well, this one has been big for a while now

Reply:
done, the script I created doesn't work yet but I'll follow up